incubator/virtualcluster/cmd/kubectl-vc/use.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
```

Comment:
I missed reviewing the previous pr before it was merged, why did we choose to add a new cli tool when syncer and vn-agent uses  `cobra` https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/virtualcluster/cmd/syncer/app/server.go#L27 & https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/virtualcluster/cmd/vn-agent/app/server.go#L27

Reply:
I don't have strong opinion about cli tool but it will be nice to be consistent. Maybe you can explain the benefits of one over another and covert the less beneficial one.

Reply:
I added a couple notes here - https://github.com/kubernetes-sigs/multi-tenancy/pull/1188#discussion_r503426564 but to summarize.

As we dev this we might want to strive for some consistency across the components so that devs don't need to relearn and we can reuse the same packages. I'm referring to things like having a shared flags implementation cause all of the components use different flags eg `master-kubeconfig` & `kubeconfig`, while not in this cli yet I'm sure we're going to hit this eventually.

Reply:
i am ok to change to cobra. The main reason for using urfave is that the container engine world is so used. And I am much more familiar with it...😂